src/Task/Git/CommitMessage.php

```diff
@@ -164,6 +164,12 @@ private function enforceTextWidth(ContextInterface $context)
 
         $errors = [];
         $lines = preg_split('/\R/u', $commitMessage);
+        foreach ($lines as $index => $line) {
+            if (strpos($line, '#') === 0) {
+                unset($lines[$index]);
+            }
+        }
+
         $subject = rtrim($lines[0]);
```

Comment:
This line will throw a notice if index 0 is filtered away above.

Reply:
You cannot have an empty commit message right? Or are there some dark ways I don't know about ... ?